docs/guides/slowness.md

```diff
@@ -0,0 +1,230 @@
+---
+title: How to improve the performance of XCUITest-based scripts
```

Comment:
I suggest shortening this title since it will be shown on the sidebar. Maybe something like 'Diagnosing Slowness'? Worth noting there is already a document titled 'Troubleshooting'.

Reply:
changed to `Diagnosing WebDriverAgent Slowness`